platform/app/src/routes/WorkList/filtersMeta.js

```diff
@@ -1,35 +1,37 @@
+import i18n from 'i18next';
+
 const filtersMeta = [
   {
     name: 'patientName',
-    displayName: 'PatientName',
+    displayName: i18n.t('StudyList:PatientName'),
     inputType: 'Text',
     isSortable: true,
     gridCol: 4,
   },
   {
     name: 'mrn',
-    displayName: 'MRN',
+    displayName: i18n.t('StudyList:MRN'),
     inputType: 'Text',
     isSortable: true,
     gridCol: 3,
   },
   {
     name: 'studyDate',
-    displayName: 'StudyDate',
+    displayName: i18n.t('StudyList:StudyDate'),
     inputType: 'DateRange',
     isSortable: true,
     gridCol: 5,
   },
   {
     name: 'description',
-    displayName: 'StudyDescription',
+    displayName: i18n.t('StudyList:StudyDescription'),
```

Comment:
In Worklist.tsx we used `'StudyList:Description'`, so I think here we want that here too, no?

Reply:
Didn't notice the "Description" key. Thank you!